metricflow-semantics/metricflow_semantics/specs/non_additive_dimension_spec.py

```diff
@@ -85,3 +88,16 @@ def linkable_specs(self, non_additive_dimension_grain: TimeGranularity) -> Seque
                 time_granularity=ExpandedTimeGranularity.from_time_granularity(non_additive_dimension_grain),
             ),
         ) + tuple(LinklessEntitySpec.from_element_name(entity_name) for entity_name in self.window_groupings)
+
+    @property
```

Comment:
Added these to ensure we could use as close to the same logic as possible in the dataflow plan and the dataflow to subquery visitor as possible to determine what specs should be used in this node.